sphinxserve/__init__.py

```diff
@@ -112,14 +123,23 @@ def serve(self):
             port,
             self.render_ev
         )
```

Comment:
Earleir, the logging message could not fit on a single line while complying with PEP-8 requirements; the standard response to such a style problem is to separate it onto multiple lines.  Later, I changed the message without re-condensing the line.  Sure, this could be shortened.